### PR TITLE
Misc Rule Fixes

### PR DESCRIPTION
**Pull request by ICodeMaster**
Various rules fixes from OWB rules repository.

### CODE REVIEW

File: Config/common/ai_strategy.cwt [hunk 1]
```diff
@@ -319,7 +319,7 @@ alias[ai_strategy_rule:ai_strategy] = {
 	# TODO
 	# operation = token:operation?
 	value = int
-	operation_target = enum[country_tags]
+	operation_target = scope[country]
 	operation_target = scope[country]
```
Comment (Yard1) [inline Config/common/ai_strategy.cwt:323]: Duplicate

File: Config/common/ai_strategy.cwt [hunk 2]
```diff
@@ -292,14 +292,14 @@ alias[ai_strategy_rule:ai_strategy] = {
...
-	id = enum[country_tags]
+	id = scope[country]
 }
 alias[ai_strategy_rule:ai_strategy] = {
 	type = operative_mission
 	mission = enum[operative_missions]
 	value = int
 	## cardinality = 0..1
-	mission_target = enum[country_tags]
+	mission_target = scope[country]
```
Comment (Yard1) [inline Config/common/ai_strategy.cwt:302]: Duplicate below

**Review by Yard1 — CHANGES_REQUESTED** [review]

File: Config/common/ai_strategy.cwt (hunk 2 above)
Comment (ICodeMaster) [reply Config/common/ai_strategy.cwt:302]: Resolved in 61931f5

File: Config/common/ai_strategy.cwt (hunk 1 above)
Comment (ICodeMaster) [reply Config/common/ai_strategy.cwt]: Resolved in 61931f5

**Comment by Yard1** [comment]
@ICodeMaster Question - why is `enum[country_tags]` removed?

**Pull request merged**